h5py/h5p.pyx

```diff
@@ -114,10 +114,7 @@ OBJECT_CREATE = lockcls(H5P_OBJECT_CREATE)
 CRT_ORDER_TRACKED = H5P_CRT_ORDER_TRACKED
 CRT_ORDER_INDEXED = H5P_CRT_ORDER_INDEXED
 
-DEFAULT = None   # In the HDF5 header files this is actually 0, which is an
-                 # invalid identifier.  The new strategy for default options
-                 # is to make them all None, to better match the Python style
-                 # for keyword arguments.
+DEFAULT = H5P_DEFAULT
```

Comment:
I don't think we need to change this for this PR - look at `set_fapl_family` for an example of how to accept a property list with a default.

If you want to change it anyway, that should be a separate PR.